src/components/confetti/index.js

```diff
@@ -12,11 +12,15 @@ const confettiStyles = () => ({
     justifyContent: 'center',
   },
   text: {
-    fontSize: '30px',
+    fontSize: '40px',
+    fontWeight: 300,
+  },
+  notifie: {
+    fontSize: '25px',
```

Comment:
Could you make the font of this class a little more lighter to differentiate if from the "Swap successful" text?